src/main/scala/mesosphere/marathon/core/launcher/LauncherModule.scala

```diff
@@ -13,13 +13,12 @@ import mesosphere.marathon.metrics.Metrics
   * and actually launching the matched tasks.
   */
 class LauncherModule(
-    clock: Clock,
     metrics: Metrics,
     conf: MarathonConf,
     taskCreationHandler: InstanceCreationHandler,
     marathonSchedulerDriverHolder: MarathonSchedulerDriverHolder,
     offerMatcher: OfferMatcher,
-    pluginManager: PluginManager) {
+    pluginManager: PluginManager)(implicit clock: Clock) {
```

Comment:
Having an implicit clock let's me more easily use it in utility methods declared in companion objects